packages/shared/src/utils/contract/contract.tsx

```diff
@@ -29,15 +29,67 @@ export const DELAY_TIME_1S_SYMBOL = 500;
 // generation_interval will be provided via API later to help us distinguish between 1-second and 2-second symbols
 export const symbols_2s = ['R_10', 'R_25', 'R_50', 'R_75', 'R_100'];
 
-export const TURBOS = {
-    LONG: 'turboslong',
-    SHORT: 'turbosshort',
+export const CONTRACT_TYPES = {
+    ACCUMULATOR: 'ACCU',
+    ASIAN: { UP: 'ASIANU', DOWN: 'ASIAND' },
+    CALL_PUT_SPREAD: { CALL: 'CALLSPREAD', PUT: 'PUTSPREAD' },
+    END: { IN: 'EXPIRYRANGE', OUT: 'EXPIRYMISS' },
+    EVEN_ODD: { ODD: 'DIGITODD', EVEN: 'DIGITEVEN' },
+    HIGH_LOW: { CALL: 'CALL', PUT: 'PUT' },
+    LB_HIGH_LOW: 'LBHIGHLOW',
+    LB_CALL: 'LBFLOATCALL',
+    LB_PUT: 'LBFLOATPUT',
+    MATCH_DIFF: { MATCH: 'DIGITMATCH', DIFF: 'DIGITDIFF' },
+    MULTIPLIER: {
+        UP: 'MULTUP',
+        DOWN: 'MULTDOWN',
+    },
+    OVER_UNDER: { OVER: 'DIGITOVER', UNDER: 'DIGITUNDER' },
+    RESET: { CALL: 'RESETCALL', PUT: 'RESETPUT' },
+    RISE_FALL: { CALL: 'CALL', PUT: 'PUT' },
+    RISE_FALL_EQUAL: { CALLE: 'CALLE', PUTE: 'PUTE' },
```

Comment:
@kate-deriv `CALLE` means `Rise Equal` and `PUTE` means `Put Equal`.
I can see that in this config, there are constants with the same string value.
For example, the constant CONTRACT_TYPES.HIGH_LOW.CALL has the same value as CONTRACT_TYPES.RISE_FALL.CALL.
Let's make this config simple and having 1 purpose to make constants for every string value of any contract type like this:
```
export const CONTRACT_TYPES = {
    // ...
    CALL: 'CALL',
    CALLE: 'CALLE',
    // ...
    PUT: 'PUT',
    PUTE: 'PUTE',
}
```
in the end, we will use CONTRACT_TYPES.CALL to replace 'CALL' string in the code. At that point, we won't need to know if it's related to rise/fall or higher/lower :)

Reply:
Completely reasonable, thank you